### DISCUSSION TITLE
请教各位同学 DolphinScheduler 伪集群安装好后执行流程任务一直显示失败是什么原因，要如何解决？

### DISCUSSION THREAD

**Discussion by godcheese** [apache/dolphinscheduler#9592] [Q&A]
请教各位同学 DolphinScheduler 伪集群安装好后执行流程任务一直显示失败是什么原因，要如何解决？是我的操作问题嘛？我搜集了以下信息以求助各位。

- CentOS 7.9
- Apache DolphinSchduler 2.0.5

**Comment by godcheese** [comment]
官方文档，issues 都查过了，谷歌、百度、stackoverflow 也查过来没搜到有相关的解决方法。

**Comment by SnowMoon-Dev** [comment]
1. please check your node health  (monitor  -> server manage
2. update to the latest version

**Reply by godcheese** [reply]
感谢已解决该问题。

**Reply by cheny5100** [reply]
怎么解决的，我的也是这个问题
